lib/FileCard/styles.scss

```diff
@@ -0,0 +1,123 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$field-card-width: 200px !default;
+$field-card-height: 180px !default;
+$field-card-action-button-size: 32px !default;
+$field-card-margin: $layout-spacing-base !default;
+
+$field-card-label-font-size: $typo-size-slight !default;
+$field-card-actions-padding: .5rem !default;
+
+$field-card-label-bg: rgba(0, 0, 0, .7) !default;
+$field-card-highlight-color: rgb(245, 218, 116);
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.file-card {
+  width: $field-card-width;
+  height: $field-card-height;
+  margin-right: $field-card-margin;
+  position: relative;
+  display: inline-block;
+  overflow: hidden;
+
+  &:hover,
+  .file-card__action:focus {
+    @for $i from 1 through 4 {
+      .file-card__action:nth-child(#{$i}) {
+        transform: translateY(0);
+        transition-delay: #{$i/20}s;
+      }
+    }
+  }
+
+  &.has-comments {
```

Comment:
I can dig that 👍 while the new state really does look like a simple state change (from non highlighted to highlighted), the underlying premised is that it received comments.

But this also leads me to an interesting discussion 😄 if we apply something like `file-card--highlight` to the state in which it has comments, and the design changes from something totally different than being highlighted, then the modifier name stops making sense... if it had been `file-card--with-comments`, then the modifier name would still make sense regardless of the visual styling.

Of course this is hardly the deepest CS issue 😂 but thought it would be worth trowing it into the pan and think about the pattern!